### PR TITLE
Block-Editor: Add tooltip on replace image's URL

### LINKED ISSUE #32249: Image block: Replace Current media URL field too small
## Description
When editing an image's URL using the **Replace** tool bar option the **Current media URL** field is far too narrow to be of much use.

## Step-by-step reproduction instructions

Please list the steps needed to reproduce the bug. For example:
1. Drag an image from another browser window.
2. Choose Replace from the image block toolbar
3. Notice that the field is far too narrow to be usable. 

## Expected behaviour
The Current media URL field should at least try to be wide enough to display half of the URL.

## Actual behaviour
You can only see approximately 16 characters. 

## Screenshots or screen recording (optional)
In this screen capture the image URL is `https://herbmiller.me/wp-content/uploads/2021/05/image-1024x513.png`

image

## WordPress information
- WordPress version: 5.7.2
- Gutenberg version: 10.6.2 
- Are all plugins except Gutenberg deactivated? Yes 
- Are you using a default theme (e.g. Twenty Twenty-One)? TT1 blocks 

## Device information
- Device: Desktop 
- Operating system: Windows 10
- Browser: Chrome Version 90.0.4430.212 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by FilipposZ**
Fixes [#32249](https://github.com/WordPress/gutenberg/issues/32249)

## What?
Added a hover tooltip when editing an image's URL using the `Replace` toolbar option.

## Why?
When editing an image's URL, being able to see the full URL is very important. However, the `Current media URL` field is far too narrow.

## How?
Added a tooltip when hovering on the image's URL field displaying the full URL.

## Testing Instructions
1. Create a new post or edit an existing post
2. Add "Image" block and select "Insert from URL" option
3. Enter an image URL
4. Select the "Replace" button
5. Hover over the URL. A tooltip should appear underneath displaying the full URL of the image

## Screenshot
Screenshot 2022-06-02 at 15 57 06

### CODE REVIEW

**Comment by FilipposZ** [comment]
Maybe @fotisps would like to give some feedback as well.

**Review by draganescu — APPROVED** [review]
I think this makes a lot of sense! 👏🏻 I would wait for some design feedback before merging.

**Comment by fotisps** [comment]
> Maybe @fotisps would like to give some feedback as well.

 Nice one @FilipposZ ! Thanks for contributing! It's ok by me!

**Comment by richtabor** [comment]
This is nice - good stuff @FilipposZ & @fotisps 👏

**Comment by cbravobernal** [comment]
Props also to @bangank36 that helped on this PR on WCEU 2022 contributor's day! 🎉 🎉

**Pull request merged**
